servers/rendering/rendering_device_binds.h

```diff
@@ -696,7 +696,8 @@ class RDPipelineColorBlendState : public RefCounted {
 	RD_SETGET(Color, blend_constant)
 
 	void set_attachments(const TypedArray<RDPipelineColorBlendStateAttachment> &p_attachments) {
-		attachments.push_back(p_attachments);
+		attachments.clear();
+		attachments.append_array(p_attachments);
```

Comment:
```suggestion
		attachments = p_attachments;
```

`is_same(attachments, p_attachments)` can be `true`, the code will not work in this case.